lib/process-services/task-list/components/task-header.component.spec.ts

```diff
@@ -141,13 +142,23 @@ describe('TaskHeaderComponent', () => {
             expect(claimButton.nativeElement.innerText).toBe('ADF_TASK_LIST.DETAILS.BUTTON.CLAIM');
         });
 
-        it('should display the claim button to the invovled group', () => {
+        it('should display the claim button if the task has invovled group/people', () => {
             component.taskDetails = new TaskDetailsModel(taskDetailsWithOutAssigneeMock);
```

Comment:
Can we change it in 
if the task is claimable?
In addition can we check, the value of the method isClaimable to be true?